test/unit-test/tests/utilities/test_state_converters.pf

```diff
@@ -0,0 +1,27 @@
+@test
+subroutine test_temp_to_potential_temp()
+    use funit
+    use state_converters, only : temp_to_potential_temp_run
+    use ccpp_kinds, only: kind_phys
+
+    integer, parameter :: ncol = 5
+    integer, parameter :: nz   = 5
+
+    real(kind_phys) :: temp(ncol, nz)
+    real(kind_phys) :: exner(ncol, nz)
+    real(kind_phys) :: theta(ncol, nz)
+    character(len=512) :: errmsg
+    integer :: errflg
+
+    temp = 1
+    exner = 1
+    theta = 1
+
+    errmsg = ""
+    errflg = 0
+
+    call temp_to_potential_temp_run(ncol, nz, temp, exner, theta, errmsg, errflg)
+
+    @assertEqual(0, errflg)
```

Comment:
I may have missed the discussion on this (perhaps from the Monday meeting?) that perhaps it may be useful to use realistic values in these kinds of tests that have known physical basis and thus the formula itself within the parameterization can also be validated in the test? It does not have to be resolved within the context of this PR so I have approved it, but it would be useful for the future. Thanks!

Reply:
Yup, you are correct!  We can add those details in here if needed (looking at @nusbaume for guidance on that one) but this was more to get the infrastructure and an example in place.  But if there's a straightforward example that we can validate with this, I'm more than happy to add it as a part of this PR!

Reply:
So is this change okay as is and then we can work on a future PR to bring in realistic values?

Reply:
Sounds good to me!